nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/bruker/BrukerData.java

```diff
@@ -1818,6 +1818,11 @@ public boolean isFrequencyDim(int iDim) {
         return result;
     }
 
+    @Override
+    public boolean isFID() {
+        return !isFrequencyDim(0);
+    }
+
```

Comment:
Not sure if this is an okay way to tell whether bruker data is an FID, just following how NMRViewData did it here https://github.com/nanalysis/nmrfx/blob/3b332a81a2608a48ba937cb58e138df493ede4d5/nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/nmrview/NMRViewData.java#L375